src/elements/form/FormUtils.js

```diff
@@ -91,9 +90,7 @@ export class FormUtils {
         }
 
         // Overrides
-        if (type === 'checkbox' && field.options) {
```

Comment:
@jgodi There is no setter in this code block that sets `type` equal to 'checkbox', so this check would never be true.